config/checkstyle/checkstyle-xpath-suppressions.xml

```diff
@@ -1,45 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Why is this file removed?  The reason it exists is so that we do not use a custom checkstyle.xml (See #3983's notes).  The goal is to use the stock checkstyle.xml that we can just update over time.  Checkstyle does make breaking DTD changes and having to navigate those is a pain.  It's easier just to drop in the published Google checkstyle.xml file every time.  Any customizations we make are stored in the suppressions file(s) which don't change format that often.